spec/features/work_packages/details/relations/primerized_relations_spec.rb

```diff
@@ -251,6 +252,49 @@ def label_for_relation_type(relation_type)
       # Bumped by one
       tabs.expect_counter("relations", 5)
     end
+
+    it "doesn't autocomplete parent, children, and WP itself" do
+      scroll_to_element relations_panel
+
+      relations_panel.find("[data-test-selector='new-relation-action-menu']").click
```

Comment:
We have helpers to check for test selectors. `page.find_test_selector("new-relation-action-menu").click`